### ISSUE TITLE
Handle Move error

### ISSUE THREAD

**Issue by neilmispelaar** [imd1005-web-development-winter-2023/group-project-digitalists#6]
# Issue

## Description

Hi @doua000 

The big issue you were having is in `handleMove()` you were only acting if the element was a "P" tag however there were no P tags on the board so nothing was happening. 

The `draw()` function writes the following code so we can update the `handleMove()` function to only run it's logic IF the element is a DIV

```html
 <div class="grid">
   <div class="tile"></div>
   <div class="tile"></div>
   <div class="tile"></div>
   <div class="tile"></div>
   <div class="tile"></div>
   <div class="tile"></div>
   <div class="tile"></div>
   <div class="tile"></div>
   <div class="tile"></div>  
</div>
```

https://github.com/imd1005-web-development-winter-2023/group-project-digitalists/blob/handle-player-click/maincopy.js

https://github.com/imd1005-web-development-winter-2023/group-project-digitalists/blob/876c8ef8ee0640a3dbf21d921c2ac6df4e0223f2/maincopy.js#L193-L204

https://github.com/imd1005-web-development-winter-2023/group-project-digitalists/blob/876c8ef8ee0640a3dbf21d921c2ac6df4e0223f2/maincopy.js#L228-L232

https://github.com/imd1005-web-development-winter-2023/group-project-digitalists/blob/876c8ef8ee0640a3dbf21d921c2ac6df4e0223f2/maincopy.js#L246-L252

## Screenshot

<img width="1536" alt="Screenshot 2023-04-08 at 12 35 57 AM" src="https://user-images.githubusercontent.com/9325038/230703317-5541bc7c-39ec-43a2-b6db-b381ad73be9d.png">

**Comment by doua000** [comment]
Thanks! I made the changes but I just wanted to confirm for edit 4: should I be removing
 tile.addEventListener("click", (event)=>{console.log("clickedtile", i)});?

**Comment by neilmispelaar** [comment]
Yeah, for Edit 4, you don’t need the event handler there since you have a more global click event handler

**Comment by doua000** [comment]
soo for some reason the grid is no longer showing, but i'm not sure what made that change :/

**Comment by neilmispelaar** [comment]
This happens often, to all developers. 

Here’s what you can do:

Start by checking your console logs in your browser. 

Are there any errors? 

Can you share a screenshot of the console (because I’m not infront of a computer right now) - Thanks!

**Comment by doua000** [comment]
I tried comparing the older version of the code with the one I updated and I can't seem to pinpoint the issue. This is what the page looks like when I run it: 
<img width="927" alt="Screen Shot 2023-04-09 at 5 28 05 PM" src="https://user-images.githubusercontent.com/122396252/230797329-2b0c18ad-3926-49fd-93ce-b888a7be4d41.png">

**Comment by doua000** [comment]
Update: okay I think it is an issue with html? because the grid looks like this right now 
<img width="626" alt="Screen Shot 2023-04-09 at 5 26 05 PM" src="https://user-images.githubusercontent.com/122396252/230797248-7e99fb0c-1576-4ab5-bda9-146ca7908395.png">
.

**Comment by neilmispelaar** [comment]
What does the console log say? 

(here’s how to access it) 
https://www.techonthenet.com/js/open_console_log.php

**Comment by doua000** [comment]
Hopefully I did this right, but this is what it says: <img width="555" alt="Screen Shot 2023-04-09 at 5 47 06 PM" src="https://user-images.githubusercontent.com/122396252/230798014-b4dad75c-3cef-4c75-98ec-ea5aa8867547.png">

**Comment by neilmispelaar** [comment]
AWESOME! 

You got it. 

Your program is having an error on line 226, trying to read a value called “firstChild”. 

Let’s have a look at that line and see what the problem is…

**Comment by neilmispelaar** [comment]
Here is the code below. 

Quick check: you using “getElementById(“grid”) to get the element. 

Can you check that the element has an id=“grid” on it. 

https://github.com/imd1005-web-development-winter-2023/group-project-digitalists/blob/59c6ac7cd5bd6f7af464fdc4ab1b16a6f63d09fd/maincopy.js#L223-L228

**Comment by doua000** [comment]
What we have is class="grid" , could I use getElementByClassName instead?

**Comment by neilmispelaar** [comment]
Hi @doua000 

Actually, you'd do the same thing you (or someone on your team) did on line 10 

```js
const grid = document.querySelector(".grid");
```

https://github.com/imd1005-web-development-winter-2023/group-project-digitalists/blob/1d5b1eb05acf07f094a95276fc8a1ca7f4f30cf4/maincopy.js#L10
